pom.xml

```diff
@@ -192,8 +192,7 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <release>10</release>
```

Comment:
What does the 10 mean?

Reply:
It's the Java version which shall be used. It should be 11 but somehow that doesn't work for me. See https://openjfx.io/openjfx-docs/#maven

Reply:
Won't this fail automatically for Java 8 and 9?

Reply:
I guess so. Maybe I can add a profile for this as well.

Reply:
Right now, I would like to get Java 11 working. Then we can think about what we want to support (Java 8 will have EOL in 2019 anyways, so...).